src/main/java/io/appium/java_client/TouchAction.java

```diff
@@ -118,13 +175,11 @@ public TouchAction moveTo(WebElement el) {
      * @param x change in x coordinate to move through.
      * @param y change in y coordinate to move through.
      * @return this TouchAction, for chaining.
+     * @deprecated {@link #moveTo(AbsoluteOffsetOption)} instead
      */
-    public TouchAction moveTo(int x, int y) {
-        ActionParameter action = new ActionParameter("moveTo");
-        action.addParameter("x", x);
-        action.addParameter("y", y);
-        parameterBuilder.add(action);
-        return this;
+    @Deprecated
+    public T moveTo(int x, int y) {
+        return moveTo(useAbsolute(x, y));
```

Comment:
* @param x **change** in x coordinate to move through.
       * @param y **change** in y coordinate to move through.